### PR TITLE
18255 trunk

### PR DESCRIPTION
**Pull request by ekaterinadimitrova2**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -1881,10 +1849,10 @@
       </java>
   </target>
 
-  <target name="_maybe_update_idea_to_java11plus" depends="init" unless="java.version.8">
-    <replace file="${eclipse.project.name}.iml" token="JDK_1_8" value="JDK_${ant.java.version}"/>
-    <replace file=".idea/misc.xml" token="JDK_1_8" value="JDK_${ant.java.version}"/>
-    <replace file=".idea/misc.xml" token="1.8" value="${ant.java.version}"/>
+  <target name="_maybe_update_idea_to_java11plus" depends="init">
```
Comment (jacek-lewandowski) [inline build.xml:1852]: do we need this target? maybe we can update idea files directly?

Comment (ekaterinadimitrova2) [reply build.xml:1852]: ` maybe we can update idea files directly?`
What do you mean by directly? We need that token change for when we want to  build&run things on 17

Comment (ekaterinadimitrova2) [reply build.xml:1852]: Clarified this one in a direct conversation. Jacek suggests adding any jvm options directly in the file.
Could be done in a follow up ticket
Also: they are not the same for both 11 and 17

**Comment by ekaterinadimitrova2** [comment]
Committed

**Pull request closed without merging**